packages/eslint-plugin-pf-codemods/test/rules/wizard-rename-hasBodyPadding.js

```diff
@@ -0,0 +1,24 @@
+const ruleTester = require('./ruletester');
+const rule = require('../../lib/rules/wizard-rename-hasBodyPadding');
+
+ruleTester.run("wizard-rename-hasBodyPadding", rule, {
+  valid: [
+    {
+      code: `import { Wizard } from '@patternfly/react-core'; <Wizard hasNoBodyPadding />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<Wizard hasNoBodyPadding />`,
```

Comment:
This could also just be `<Wizard hasNoPadding>`. Since there's no `import { Wizard } from '@patternfly/react-core';` it might be another Wizard implementation!